### DISCUSSION TITLE
What's new in 0.26.2?

### DISCUSSION THREAD

**Discussion by Oobiewan** [redhat-developer/vscode-xml#953] [Q&A]
I'm on the pre-release channel and VS Code updated this plugin multiple times in the last couple of months, currently I'm at v0.26.2023111808. Where can I find what has changed?
It is kind of unsettling that the latest release is 0.26.1 in this project, how do I know that I'm not installing something malicious by a third party?

**Comment by fbricon** [comment]
Targeting the 0.27.0 release, the latest 0.26.timestamp pre-release build contain these fixes : https://github.com/redhat-developer/vscode-xml/milestone/36?closed=1

The builds are published by Red Hat to the VS Code Marketplace. If you don't trust them you can always uninstall the extension.
